BurnerB/FreeMentors-V2
# #168365442 Admin make user mentor

#### What does this PR do?
Allows an Admin to make  a user into a mentor
#### Description of Task to be completed?
* Create the make mentor endpoint
* Ensure tests pass
#### How should this be manually tested?
* Clone this  repository https://github.com/BurnerB/FreeMentors-V2.git and cd into FreeMentors-V2
* checkout to branch ft-Admin-make-user-mentor-168365442
* Run npm --init to install all dependancies.
* Run command npm test on your terminal
* Tests should pass
#### What are the relevant pivotal tracker stories?
[#168365442](https://www.pivotaltracker.com/story/show/168365442)
#### Screenshots (if appropriate) 
![admin make mentor](https://user-images.githubusercontent.com/45785786/64549744-02336f80-d33a-11e9-84fa-5dc1cc23f5de.png)
